### PR TITLE
OH-27 and OH-50; +button color; +image on decks

### PR DESCRIPTION
**Pull request by Logotipiwe**

### CODE REVIEW

**Comment by IAMDOKTOR** [comment]
it would be nice to add description to commits. Please consider doing so in future PRs. Thank you!

**Comment by Logotipiwe** [comment]
image

File: deploy/haur.local.test.yml
Comment (IAMDOKTOR) [inline deploy/haur.local.test.yml:1]: I'm not certain on the reason behind this change. Could you please specify?

**Review by IAMDOKTOR — COMMENTED** [review]
looking good

**Pull request merged**